src/Illuminate/Auth/CreatesUserProviders.php

```diff
@@ -33,16 +33,13 @@ public function createUserProvider($provider = null)
             );
         }
 
-        switch ($driver) {
-            case 'database':
-                return $this->createDatabaseProvider($config);
-            case 'eloquent':
-                return $this->createEloquentProvider($config);
-            default:
-                throw new InvalidArgumentException(
-                    "Authentication user provider [{$driver}] is not defined."
-                );
-        }
+        return match ($driver) {
+            'database' => $this->createDatabaseProvider($config),
+            'eloquent' => $this->createEloquentProvider($config),
+            default => (throw new InvalidArgumentException(
```

Comment:
Why is the throw wrapped in parentheses?

Reply:
There are not necessary. I removed them.